Application throws exception immediately at runtime

Upon running the jar file of the latest release, the application immediately throws the following exception. This prevents the entire application from being usable.
```
Nov 02, 2023 12:55:54 AM wedlog.address.ui.UiManager start
SEVERE: Invalid URL: Invalid URL or resource not found
java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found
        at javafx.scene.image.Image.validateUrl(Image.java:1130)
        at javafx.scene.image.Image.<init>(Image.java:624)
        at wedlog.address.ui.MainWindow.fillInnerParts(MainWindow.java:136)
        at wedlog.address.ui.UiManager.start(UiManager.java:45)
        at wedlog.address.MainApp.start(MainApp.java:174)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: Invalid URL or resource not found
        at javafx.scene.image.Image.validateUrl(Image.java:1119)
        ... 13 more

Nov 02, 2023 12:55:54 AM wedlog.address.ui.UiManager showFatalErrorDialogAndShutdown
SEVERE: Fatal error during initializing Invalid URL: Invalid URL or resource not foundInvalid URL: Invalid URL or resource not found
java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found
        at javafx.scene.image.Image.validateUrl(Image.java:1130)
        at javafx.scene.image.Image.<init>(Image.java:624)
        at wedlog.address.ui.MainWindow.fillInnerParts(MainWindow.java:136)
        at wedlog.address.ui.UiManager.start(UiManager.java:45)
        at wedlog.address.MainApp.start(MainApp.java:174)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: Invalid URL or resource not found
        at javafx.scene.image.Image.validateUrl(Image.java:1119)
        ... 13 more
```

Version: [v1.3(trial)](https://github.com/AY2324S1-CS2103T-F11-2/tp/releases/tag/v1.3.trial)

--- a/src/main/java/wedlog/address/MainApp.java
+++ b/src/main/java/wedlog/address/MainApp.java
@@ -36,7 +36,7 @@ import wedlog.address.ui.UiManager;
  */
 public class MainApp extends Application {
 
-    public static final Version VERSION = new Version(1, 2, 0, false);
+    public static final Version VERSION = new Version(1, 3, 0, false);
 
     private static final Logger logger = LogsCenter.getLogger(MainApp.class);
 

--- a/src/main/java/wedlog/address/ui/MainWindow.java
+++ b/src/main/java/wedlog/address/ui/MainWindow.java
@@ -6,11 +6,13 @@ import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.control.MenuItem;
 import javafx.scene.control.TextInputControl;
+import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
 import javafx.scene.input.KeyCombination;
 import javafx.scene.input.KeyEvent;
 import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
+import wedlog.address.MainApp;
 import wedlog.address.commons.core.GuiSettings;
 import wedlog.address.commons.core.LogsCenter;
 import wedlog.address.logic.Logic;
@@ -25,6 +27,7 @@ import wedlog.address.logic.parser.exceptions.ParseException;
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static final String LOGO_APPLICATION = "/images/WedLogLogo.png";
 
     private final Logger logger = LogsCenter.getLogger(getClass());
 
@@ -130,7 +133,7 @@ public class MainWindow extends UiPart<Stage> {
      */
     void fillInnerParts() {
 
-        logoImage.setImage(new javafx.scene.image.Image("./images/WedLogLogo.png"));
+        logoImage.setImage(getImage(LOGO_APPLICATION));
 
         setGuestListPanel();
         setVendorListPanel();
@@ -250,5 +253,9 @@ public class MainWindow extends UiPart<Stage> {
         dietaryPlaceholder.getChildren().clear();
         dietaryPanel = new DietaryPanel(logic);
         dietaryPlaceholder.getChildren().add(dietaryPanel.getRoot());
+    }
+
+    private Image getImage(String imagePath) {
+        return new Image(MainApp.class.getResourceAsStream(imagePath));
     }
 }

